src/index.ts

```diff
@@ -0,0 +1,286 @@
+  ListResourcesRequestSchema,
+  ReadResourceRequestSchema,
+} from "@modelcontextprotocol/sdk/types.js"
+
+// Import tools
+import * as browserEval from "./tools/browser-eval.js"
+import * as enableCacheComponents from "./tools/enable-cache-components.js"
+import * as init from "./tools/init.js"
+import * as nextjsDocs from "./tools/nextjs-docs.js"
+import * as nextjsRuntime from "./tools/nextjs-runtime.js"
+import * as upgradeNextjs16 from "./tools/upgrade-nextjs-16.js"
+
+// Import prompts
+import * as upgradeNextjs16Prompt from "./prompts/upgrade-nextjs-16.js"
+import * as enableCacheComponentsPrompt from "./prompts/enable-cache-components.js"
+
+// Import resources
+import * as cacheComponentsOverview from "./resources/(cache-components)/overview.js"
+import * as cacheComponentsCoreMechanics from "./resources/(cache-components)/core-mechanics.js"
+import * as cacheComponentsPublicCaches from "./resources/(cache-components)/public-caches.js"
+import * as cacheComponentsPrivateCaches from "./resources/(cache-components)/private-caches.js"
+import * as cacheComponentsRuntimePrefetching from "./resources/(cache-components)/runtime-prefetching.js"
+import * as cacheComponentsRequestApis from "./resources/(cache-components)/request-apis.js"
+import * as cacheComponentsCacheInvalidation from "./resources/(cache-components)/cache-invalidation.js"
+import * as cacheComponentsAdvancedPatterns from "./resources/(cache-components)/advanced-patterns.js"
+import * as cacheComponentsBuildBehavior from "./resources/(cache-components)/build-behavior.js"
+import * as cacheComponentsErrorPatterns from "./resources/(cache-components)/error-patterns.js"
+import * as cacheComponentsTestPatterns from "./resources/(cache-components)/test-patterns.js"
+import * as cacheComponentsReference from "./resources/(cache-components)/reference.js"
+import * as nextjsFundamentalsUseClient from "./resources/(nextjs-fundamentals)/use-client.js"
+import * as nextjs16BetaToStable from "./resources/(nextjs16)/migration/beta-to-stable.js"
+import * as nextjs16Examples from "./resources/(nextjs16)/migration/examples.js"
+
+// Tool registry
+const tools = [
+  browserEval,
+  enableCacheComponents,
+  init,
+  nextjsDocs,
+  nextjsRuntime,
+  upgradeNextjs16,
+]
+
+// Prompt registry
+const prompts = [upgradeNextjs16Prompt, enableCacheComponentsPrompt]
+
+// Resource registry
+const resources = [
+  cacheComponentsOverview,
+  cacheComponentsCoreMechanics,
+  cacheComponentsPublicCaches,
+  cacheComponentsPrivateCaches,
+  cacheComponentsRuntimePrefetching,
+  cacheComponentsRequestApis,
+  cacheComponentsCacheInvalidation,
+  cacheComponentsAdvancedPatterns,
+  cacheComponentsBuildBehavior,
+  cacheComponentsErrorPatterns,
+  cacheComponentsTestPatterns,
+  cacheComponentsReference,
+  nextjsFundamentalsUseClient,
+  nextjs16BetaToStable,
+  nextjs16Examples,
+]
+
+// Create server
+const server = new Server(
+  {
+    name: "next-devtools-mcp",
+    version: "0.2.6",
```

Comment:
can we load package.json version here so we don't need to manually update each time